west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 85a0034809612b332fbdf478357e8a4109c493fd
+      revision: pull/844/head
```

Comment:
Nothing wrong with this line, but the commit message is not formatted correctly.

The detailed message says

```
    -This enables TF-M isolation level 2 in TF-M
     tests.
```

which is really just repeating the shortlog. Why is this change being made?

For a quick overview of what is expected, please see here: https://www.youtube.com/watch?v=mnVNLM7ynMQ&t=771s

Reply:
Removed the - and updated the description here.